frontend/src/views/Abfrage.vue

```diff
@@ -521,51 +526,49 @@ export default class Abfrage extends Mixins(
 
   private async saveAbfrage(): Promise<void> {
     if (this.validate()) {
-      this.saveInfrastrukturabfrage(true);
+      this.saveBauleitplanverfahren(true);
     } else {
       this.showWarningInInformationList("Es gibt noch Validierungsfehler");
     }
   }
 
-  private async saveInfrastrukturabfrage(showToast: boolean): Promise<void> {
-    const validationMessage: string | null = this.findFaultInInfrastrukturabfrageForSave(this.abfrage);
+  private async saveBauleitplanverfahren(showToast: boolean): Promise<void> {
```

Comment:
Die Methodenbenennung und Parameter in dieser View an der Basisklasse AbfrageAngelegtDto, AbfrageInBearbeitungSachbearbeitungDto, AbfrageInBearbeitungFachreferatDto bzw. AbfrageDto ausrichten, da ein Abfrage...Dto anstelle eines konkreten Dto gespeichert wird.

Es wird kein BauleitplanverfahrenDto gespeichert

Reply:
wie besprochen habe ich _saveAbfrage_ umgestellt.